Solutions/FIT5032_Week04_DatabaseFirst/FIT5032_W4_DatabaseFirst/Views/Units/Index.cshtml

```diff
@@ -36,9 +36,9 @@
             @Html.DisplayFor(modelItem => item.Description)
         </td>
         <td>
-            @Html.ActionLink("Edit", "Edit", new { id=item.Id }) |
-            @Html.ActionLink("Details", "Details", new { id=item.Id }) |
-            @Html.ActionLink("Delete", "Delete", new { id=item.Id })
+            @Html.ActionLink("Edit", "Edit", new { id=item.Id }, new { @class = "btn btn-default", @style = "width: 80px" }) 
```

Comment:
Hi, @sprintal 

Please avoid using inline styles. When you do an @style it would create an inline style. Normally, it is a good idea to avoid inline styles. It would be much better to not use inline style in this scenario. 

Refer to 
[Quora](https://www.quora.com/Whats-the-best-practice-inline-internal-or-external-CSS)
[MDN Link](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/CSS_and_JavaScript)

Regards,

Jian

Reply:
Thanks for the suggestion, I will revise accordingly.